### ISSUE TITLE
Arguments mismatch

### ISSUE THREAD

**Issue by kathir-t** [shashwat001/whatsapp-web-graph#4]
**The error:**

> [2021-06-15 22:54:29,341] {_logging.py:61} - error from callback <function WhatsApp.connect.<locals>.<lambda> at 0x7f9e55f82700>: on_close() takes 1 positional argument but 2 were given

**The cause:**

In [Client.py L292](https://github.com/shashwat001/whatsapp-web-graph/blob/863b1964ea9384d9daf202822cafec36c252c962/client.py#L292) , `ws` is passed as a parameter to the `on_close()` method, whilst the actual method does not need it.

**Comment by shashwat001** [comment]
You are right, but need to add ws to the function argument, as python library passes it while calling.
